docs/realm.js

```diff
@@ -371,6 +371,12 @@ class Realm {
  *   will be skipped if another process is accessing it.
  * @property {string} [path={@link Realm.defaultPath}] - The path to the file where the
  *   Realm database should be stored.
+ * @property {string} [fifoFilesFallbackPath] - Opening a Realm creates a number of FIFO files in order to
```

Comment:
Perhaps we don't need to expose the internal details in particular in the naming, but suffice to have a generically named admin path?
perhaps `writableFallbackPath`? Or just `writablePath`?

Reply:
Normally I would agree, but in this case, it is directly related to the files being FIFO's. Also, this option should be rarely used, so making it as descriptive as possible is actually desirable IMO.